lib/babe/verify_test.go

```diff
@@ -648,13 +647,14 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 			name:     "valid digest items, getAuthorityIndex error",
 			verifier: *babeVerifier5,
 			header:   header7,
-			expErr:   errors.New("failed to get authority index no digest provided"),
+			expErr: fmt.Errorf("could not verify block equivocation: failed to get authority index for block %s: for block hash %s: %w", //nolint:lll
+				h, types.NewEmptyHeader().Hash(), errNoDigest),
 		},
 		{
 			name:     "get header err",
 			verifier: *babeVerifier6,
 			header:   header7,
-			expErr:   errors.New("failed get header get header error"),
+			expErr:   fmt.Errorf("could not verify block equivocation: failed to get header for block %s: get header error", h),
```

Comment:
expecting constant string since there is no need to have a dynamic expectation.

```suggestion
			expErr: fmt.Errorf("could not verify block equivocation: " +
				"failed to get header for block 0x0100000000000000000000000000000000000000000000000000000000000000: " +
				"get header error"),
```